templates/patterns/list_item/list_item.ui_patterns.yml

```diff
@@ -23,6 +20,18 @@ list_item:
     thumbnail_secondary:
       label: "List item: thumbnail with secondary image"
       description: "Thumbnail list item, contains a secondary image, metadata information and a linked title."
+    date:
+      label: "List item: event"
```

Comment:
Its weird to call this pattern "event" since it doesn't necessarily involve an event.

Reply:
Probably it's because it is used on an event listing item at the moment.

Reply:
You are right @imanoleguskiza I'll change their name to past date, upcoming date etc.

Reply:
Still ongoing date doesn't make much sense :D

Reply:
Actually you are right, the date blocks are just for events: https://ec.europa.eu/component-library/ec/components/date-block/usage/

Reply:
We agreed with @imanoleguskiza to keep it event. resolving.